woocommerce/woocommerce
# [7.0] Fix mobile messaging

### All Submissions:






### Changes proposed in this Pull Request:

This PR makes mobile messaging to be added only for `WC_Email_New_Order` type of email.



Closes #35016 .

### How to test the changes in this Pull Request:

(copied and updated from the issue)

1. Place an order with 7.0, see rules for Email links/Order types in https://github.com/woocommerce/woocommerce/pull/34467
2. Check footer links in Merchant New Order Emails. Assert it does contain any mobile messages
3. Check Customer "Your order has been received!" Email. Assert it does not contain any mobile messages

### Other information:




### FOR PR REVIEWER ONLY:
